benchmarks/kubectl-mtb/internal/kubectl-mtb/run.go

```diff
@@ -160,7 +160,7 @@ func validateFlags(cmd *cobra.Command) error {
 
 	_, err = benchmarkRunOptions.ClusterAdminClient.CoreV1().Namespaces().Get(context.TODO(), benchmarkRunOptions.TenantNamespace, metav1.GetOptions{})
 	if err != nil {
-		return fmt.Errorf("invalid namespace")
+		return err
```

Comment:
If you have no connectivity to the cluster, this is the first API call made which means you get a confusing "invalid namespace" error instead of the true error, which is something like "i/o timeout".

Even when the namespace doesn't exist, the message is "namespace 'foo' not found"